packages/client/src/classes/client.js

```diff
@@ -54,6 +54,12 @@ class Client {
    * Set default header
    */
   setDefaultHeader(key, value) {
+    if (key !== null && typeof key === 'object') {
+      // key is an object
+      Object.assign(this.defaultHeaders, key);
+      return this;
```

Comment:
could just return `Object.assign` instead of returning `this` but either way is good by me.